basic/azure-ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     steps:
       - task: NodeTool@0
         inputs:
-          versionSpec: '12.x'
+          versionSpec: '18.x'
```

Comment:
Cypress v12 dropped node 12 support. This is an example for users. Updated to use the latest LTS node version.